docs/4.1/aws_oss_guide.md

```diff
@@ -124,6 +124,63 @@ proxy and provides the TCP connections needed for Teleport proxy SSH connections
 IAM is the recommended tool for creating service access. This guide will follow the 
 best practice of principle of least privilege (PoLP). 
 
+#### IAM for Amazon S3
+
+In order to grant an IAM user in your AWS account access to one of your buckets, `example.s3.bucket` you will need to grant the following permissions: `s3:ListBucket`, `s3:ListBucketVersions`, `s3:PutObject`, `s3:GetObject`, `s3:GetObjectVersion`
+
+An example policy is shown below:
+
+```
+{
+   "Version": "2012-10-17",
+   "Statement": [
+     {
+       "Effect": "Allow",
+       "Action": [
+         "s3:ListBucket",
+         "s3:ListBucketVersions"
+        ],
+       "Resource": ["arn:aws:s3:::example.s3.bucket"]
+     },
+     {
+       "Effect": "Allow",
+       "Action": [
+         "s3:PutObject",
+         "s3:GetObject",
+         "s3:GetObjectVersion"
+       ],
+       "Resource": ["arn:aws:s3:::example.s3.bucket/*"]
+     }
+   ]
+ }
+```
+
+#### IAM for DynamoDB
+
+In order to grant an IAM user access to DynamoDB make sure that the IAM role assigned to Teleport is configured with proper permissions. 
+
+An example policy is shown below:
```

Comment:
I would be inclined to add a not that `u-west-1:123456789012:table/prod.teleport.auth` should be updated with the customers dynamoDB instance.